placementApp/serializers.py

```diff
@@ -66,6 +66,12 @@ class Meta:
         model = Company
         fields = "__all__"
 
+class CoordinatorSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Coordinator
+        fields = "__all__"
```

Comment:
I think the fields such as password, date_joined, last_login, is_admin, is_active, is_staff, is_superuser should be excluded